mvp-spec/operator-api.md

```diff
@@ -16,13 +16,13 @@ These endpoints need to support both the "3PC" and "no 3PC" contexts.
-| Get identity                            | -                       | list of:<br>public key + start and end dates if any   | Get the operator's "key" to use it for verifying an ID signed by this operator                                                                                                            | This is used by websites to get the operator identity to verify signatures of preferences and id.<br>Also used by audits.                                                                                                                                                               | GET /v1/json/identity       | Not available                   |
+| Endpoint                                | Input*                  | Output                                                                     | Description                                                                                                                                                                                                                                                                    | Notes                                                                                                                                                                                                                                                                                   | Rest                   | Redirect               |
+| --------------------------------------- | ----------------------- | -------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- | ---------------------- |
+| Read data if exists, or return a new Id | -                       | List of IDs.<br>List of preferences (if any)<br>newly generated ID, if any | If ID 🍪 exist, return it.<br><br>Otherwise: create a new ID, sign it and return it. In this case, the **new ID is not stored in a cookie yet**..<br>If preferences 🍪 exist, return them<br><br>Note: when called in REST mode, a special short-life "3PC" cookie is also set | Why don't we save the newly generated ID?<br>Because haven't yet received consent. The new ID will be saved as cookie, along with preferences, when it is provided via the "Write data" endpoint ⬇️                                                                                     | GET /v1/json/read      | GET /v1/redirect/read  |
```

Comment:
Either above of the table or within the first column, it is good to remind that the output is conditioned by the session of the user. Therefore, two users has different output when reaching the same endpoint.